### PR TITLE
DF-2308

### PR DESCRIPTION
**Pull request by jimmynotjim**
Temporarily removed the records until we have records to show

## Changes

- Added "Coming Soon" heading

## Testing

- checkout branch and nav to `/sub-pages/foia-records/`

## Review

- @schaferjh 
- @anselmbradford 
- @sebworks 
- @KimberlyMunoz 

## Preview

screen shot 2015-07-30 at 2 12 59 pm

[Preview this PR without the whitespace changes](?w=0)

### CODE REVIEW

File: src/sub-pages/foia-records/index.html [hunk 1]
```diff
@@ -23,15 +23,22 @@
     </section>
     {% endif %}
 
+    {# TODO: remove when we have real records #}
+    <h2>Comming Soon</h2>
```
Comment (KimberlyMunoz) [inline src/sub-pages/foia-records/index.html]: Typo: `Coming soon`

**Comment by schaferjh** [comment]
Spelling. :)

And, so it looks less like a header for records that are due out soon, would it be possible for it to be an h4 and say "FOIA records content coming soon"?

**Comment by jimmynotjim** [comment]
Spelling mistake? I don't know what you're talking about ;)

Updated with correct spelling, full title text, h4, and changelog entry.

**Comment by KimberlyMunoz** [comment]
:+1:

**Comment by sebworks** [comment]
:+1:

**Pull request merged**